src/app/pages/main/components/sidenav/sidenav.component.ts

```diff
@@ -22,4 +24,21 @@ export class SidenavComponent {
   onCloseInfo() {
     this.closeInfo.emit();
   }
+
+  onShare() {
+    const nav = window.navigator as any;
```

Comment:
Any reason what `any` is required?

```suggestion
    const nav = window.navigator;
```